andy.json

```diff
@@ -1,11 +1,11 @@
 {
     "name": "andy",
 
-    "model": "gpt-3.5-turbo-0125",
+    "model": "claude-3-haiku-20240307",
```

Comment:
Let's not change this quite yet. If only because having an openai account is still more common.

Reply:
forgot to change it back